<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>471</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>781</width>
      <height>471</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="main">
     <widget class="QPushButton" name="goUserButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 209, 122);</string>
      </property>
      <property name="text">
       <string>Я пользователь</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goWorkerButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 209, 122);</string>
      </property>
      <property name="text">
       <string>Я работник</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ownerButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(224, 27, 36);</string>
      </property>
      <property name="text">
       <string>Я владелец</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user">
     <widget class="QPushButton" name="returnToMain">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>430</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>На главную</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>531</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    border: 1px solid #dcdcdc;
    gridline-color: #dcdcdc;
    background-color: #f9f9f9;
    alternate-background-color: #f1f1f1;
}

QHeaderView::section {
    background-color: #f0f0f0;
    color: #333;
    padding: 4px;
    border: 1px solid #dcdcdc;
}

QTableView::item {
    padding: 4px;
    border: none;
}

QTableView::item:selected {
    background-color: #a8d8ea;
    color: #333;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buyButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>390</y>
        <width>87</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Купить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="worker">
     <widget class="QPushButton" name="returnToMain_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>430</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>На главную</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewDisks">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    border: 1px solid #dcdcdc;
    gridline-color: #dcdcdc;
    background-color: #f9f9f9;
    alternate-background-color: #f1f1f1;
}

QHeaderView::section {
    background-color: #f0f0f0;
    color: #333;
    padding: 4px;
    border: 1px solid #dcdcdc;
}

QTableView::item {
    padding: 4px;
    border: none;
}

QTableView::item:selected {
    background-color: #a8d8ea;
    color: #333;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Диски в продаже</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewGenre">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>371</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    border: 1px solid #dcdcdc;
    gridline-color: #dcdcdc;
    background-color: #f9f9f9;
    alternate-background-color: #f1f1f1;
}

QHeaderView::section {
    background-color: #f0f0f0;
    color: #333;
    padding: 4px;
    border: 1px solid #dcdcdc;
}

QTableView::item {
    padding: 10px;
    border: none;
}

QTableView::item:selected {
    background-color: #a8d8ea;
    color: #333;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Все жанры</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titleEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="releaseEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>300</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="priceEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>title</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>release</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>price</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stockEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>stock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addDisk">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>400</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить диск</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addGenre">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>350</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить Жанр</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="genreEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>название жанра</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="admin">
     <widget class="QPushButton" name="returnToMain_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>430</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>На главную</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableOrders">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>531</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Все заказы</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Видеопрокат</string>
    </property>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
